docs/api.asciidoc

```diff
@@ -30,7 +32,8 @@ If you are using Rails this is done automatically for you.
 [[api-stop]]
 ==== `ElasticAPM.stop`
 
-Stop the currently running agent. Use this inside `at_exit` in your <<getting-started-rack,Rack app>> to gracefully shut down.
+Stop the currently running agent. Use this inside `at_exit` in your
+<<getting-started-rack,Rack app>> to gracefully shut down.
```

Comment:
I cannot add a comment on line 29 so I add it here:
**ElasticAPM expects a single instance of the Agent.**

Can you add a little bit more description here please. Does this mean I cannot have multiple agents in multiple threads/processes?